src/scala/io/bazel/rules_scala/scrooge_support/Compiler.scala

```diff
@@ -128,7 +128,7 @@ class Compiler {
           defaultOptional = isJava,
           skipIncludes = false,
           documentCache
-        )(com.twitter.logging.NullLogger) // scrooge warns on file names with "/"
```

Comment:
ahh, I see.. :/ I guess so, I just hope it does spit a ton of warnings out on us.